compendium/DeclarativeServices/src/manager/states/CCActiveState.hpp

```diff
@@ -78,12 +83,12 @@ class CCActiveState final
    */
   ComponentState GetValue() const override { return ComponentState::ACTIVE; }
 
-  void WaitForTransitionTask() override
-  {
-    latch.Wait();
-  }
+ 
 private:
-  CounterLatch latch;
```

Comment:
Just a question: what was "latch" used for, and what's not happening now that it was removed?